code/modules/atmospherics/he_pipes.dm

```diff
@@ -58,8 +58,10 @@
 		..()
 	else
 		var/datum/gas_mixture/pipe_air = return_air()
+		var/turf/T = get_turf(loc)
+		var/turf/turf_above = GET_TURF_ABOVE(T)
```

Comment:
Could be simplified again

Reply:
This one cannot, T is used below (istype(GET_TURF_BELOW(T), /turf/space))